src/shutdown/shutdown.c

```diff
@@ -305,6 +342,16 @@ int main(int argc, char *argv[]) {
         (void) cg_get_root_path(&cgroup);
         in_container = detect_container() > 0;
 
+        /* If the logging messages are going to KMSG, and if we are not running from a container,
+         * then try to update the sysctl kernel.printk current value in order to see "notice" messages;
+         * This current log level is not updated if already big enough.
+         */
+        if (!in_container && !IN_SET(log_get_target(), LOG_TARGET_AUTO,
+                                                       LOG_TARGET_JOURNAL_OR_KMSG,
+                                                       LOG_TARGET_SYSLOG_OR_KMSG,
+                                                       LOG_TARGET_KMSG))
+                update_sysctl_printk_log_level(LOG_NOTICE);
```

Comment:
hmm, in all our codebase the assumption is always that by default LOG_INFO messages are shown. Hence I figure this should be LOG_INFO here, too...

Also, maybe "update" → "bump" to indicate that we only increase the log level here, never decrease it. (following similar naming in src/core/main.c for various sysctls)

Reply:
Well, on Arch Linux this produce a very verbose shutdown (with LOG_INFO)...
I fear to have complaint about this...